### PR TITLE
Create Chapter 2 Figures

### PR DESCRIPTION
**Pull request by SDemirdjian**

### CODE REVIEW

File: Chapter2/Figures/Listing2_2.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+namespace Chapter2
+{
+    public class GuessStatisticsMessage
+    {
+        private string number;
+        private string verb;
+        private string pluralModifier;
+
+        public string make(char candidate, int count)
```
Comment (dianabice) [inline Chapter2/Figures/Listing2_2.cs:11]: C# coding conventions has `methods` in PascalCasing

Comment (SDemirdjian) [reply Chapter2/Figures/Listing2_2.cs:11]: Goodpoint! I was just translating with thought of "change as little as possible", but really I should be translating with "how would this have been written in C#".

**Comment by timothyFbrown** [comment]
How do you perform a code review?

**Review by timothyFbrown — COMMENTED** [review]
Could we change the language [Ook!](https://esolangs.org/wiki/Ook!) with a polynesian coding style?

**Review by dianabice — APPROVED** [review]

**Comment by timothyFbrown** [comment]
Diana how do you approve the review, I would think with two reviewers we shouldn't be able to merge a pull request?

**Comment by dianabice** [comment]
@timothyFbrown You go to the file changed section of the PR and it give you the option to start a review. I was able to merge it, it said that I was requested to review but approval wasn't necessary for the merge. I think you need to change the criteria on the repo

**Pull request merged**